exercises/book-store/canonical-data.json

```diff
@@ -147,6 +147,15 @@
            "basket": [1,1,2,2,3,3,4,5,1,1,2,2,3,3,4,5]
          },
          "expected": 10240
+        },
+        {
+         "property": "total",
+         "description": "Two groups of four and a group of five",
+         "comments": ["Suggested grouping, [[1,2,3,4],[1,2,3,5],[1,2,3,4,5]]. Solutions can pass all the other tests if they just try to group without using any groups of 5. This test case breaks that since it requires both 4-groups and 5-groups."],
+         "input": {
+           "basket": [1,1,1,2,2,2,3,3,3,4,4,5,5]
```

Comment:
As mentioned in [my comment here](https://github.com/exercism/python/pull/2141#issuecomment-557761913), I would suggest shuffling the input numbers here (or adding an additional case that has the inputs shuffled), as this ensures a more robust solution.

Reply:
Good point, I've added a second test to check for that.